examples/BlueprintPlugin/Source/PluginEditor.cpp

```diff
@@ -18,26 +18,34 @@ BlueprintPluginAudioProcessorEditor::BlueprintPluginAudioProcessorEditor (Bluepr
     File sourceDir = (File (__FILE__)).getParentDirectory();
 
     addAndMakeVisible(appRoot);
-    appRoot.evalScript(sourceDir.getChildFile("ui/build/js/main.js").loadFileAsString());
-    appRoot.enableHotkeyReloading();
-
-    appRoot.registerNativeMethod(
+    appRoot.evaluate(sourceDir.getChildFile("ui/build/js/main.js").loadFileAsString());

```

Comment:
:+1: perfect, thanks for this. You're right about the `enableHotKeyReloading` method, that was removed. Let's just delete that line here, and if you don't mind it looks like there's some extra whitespace on line 30

Reply:
ok should be done

Reply:
killed the duktape method as well